### DISCUSSION TITLE
Seconds! Defining alternate project representatives

### DISCUSSION THREAD

**Discussion by ebullient** [commonhaus/foundation#203] [Consensus Building]
[![🗳️ Vote progress](https://www.commonhaus.org/votes/commonhaus/foundation/203.svg)](https://github.com/commonhaus/foundation/discussions/203#discussioncomment-10887348 "DC_kwDOKRPTI84ApiC0")

### Proposal

We've made it pretty clear that each project gets one vote. 
What should also be clear is that we want to move away from single points of failure, too. 

I'd like to encourage all projects to have a "second" EGC rep.

In my mind, the second vote is counted only when the primary is missing. When that is the case, that vote also does count when tallying quorum.

In some sense, it then is up to the project who is the primary and who is the secondary depending on time commitments, etc. Both would have access to egc discussions and lists.

Anything I missed?

### Voting group

@commonhaus/cf-egc

### Consensus mechanism

Please use emoji reactions to indicate your support:
- **👍 (:+1:)** if it looks good to you
- **👀 (:eyes:)** if you're "ok" with it (it may not be your favorite)
- **👎 (:-1:)** if you think this needs discussion first

Ask questions or raise concerns in comments.

**Comment by kenfinnigan** [comment]
You're saying we won't track the "primary" between the 2 names?

And if both vote, we only take primary. If only secondary votes we take that one. Is that correct?

How do we handle two votes with different choices? Particularly if the secondary has concerns the primary hasn't raised?

**Reply by ebullient** [reply]
Primary has precedence.

> And if both vote, we only take primary. If only secondary votes we take that one. Is that correct?

Correct. This is (by definition) a trust exercise, but training a successor and having a backup leader is always a trust exercise, isn't it?

As far as raising concerns goes, both will be able to see and contribute to the discussion. So if there are concerns to raise, both would have the opportunity to raise them.

**Reply by kenfinnigan** [reply]
That all sounds good.

Do we need to have anything related to handling a primary voting "yes" and a secondary voting "no". Or do we just take the "primary" vote and leave it to the project to resolve their different perspectives?

**Reply by ebullient** [reply]
Tooling will handle taking the primary... and yes, the project will have to handle their differences of opinion.

**Reply by sigrunixia** [reply]
👍🏻 There is value in also showing when a primary and secondary do agree, both as a matter of cohesion, and for legal matters since these would be in written records in the case of later conflict.

[a comment by an automated account is not shown]

**Comment by JooHyukKim** [comment]
Seems like something we've discussed about? @cowtowncoder

**Reply by cowtowncoder** [reply]
Yes @JooHyukKim .

**Reply by ebullient** [reply]
Exactly. This is mechanically how I bring you in, @JooHyukKim. ;)

**Comment by ebullient** [comment]
vote::result We're in agreement. I'll get to work.
